### ISSUE TITLE
[Improvement] Deprecate usage of node-sass, instead only use of dart-sass

### ISSUE THREAD

**Issue by Divlo** [Thream/styled-jsx-plugin-sass#28]
## Proposal

We should only support `dart-sass` and not `node-sass`.

Quote from `node-sass` (see: https://www.npmjs.com/package/node-sass) :

> Warning: LibSass and Node Sass are deprecated. While they will continue to receive maintenance releases indefinitely, there are no plans to add additional features or compatibility with any new CSS or Sass features. Projects that still use it should move onto Dart Sass.

**Comment by bfanger** [comment]
The peerDependency on **node-sass**  causes the `npm install --save-dev @styled-jsx/plugin-sass` to fail.
_( MacOS 11 / node 16 / npm 7 )_

Workaround:
```
npm install --save-dev --legacy-peer-deps  @styled-jsx/plugin-sass
```

**Comment by Divlo** [comment]
Thanks for sharing the temporary workaround! @bfanger 
I published `4.0.0`, and this workaround is no longer necessary, as we're now using `dart-sass`.
Could you confirm that it actually works, with this new version ? :smile:

**Comment by bfanger** [comment]
@Divlo 4.0.0 fixed the `npm install` issue 👍
